bats/foreman_helper.bash

```diff
@@ -35,6 +35,13 @@ tSkipIfOlderThan41() {
   fi
 }
 
+tSkipIfOlderThan43() {
+  KATELLO_VERSION=$(tKatelloVersion)
+  if [[ $KATELLO_VERSION != 4.* || $KATELLO_VERSION == 4.2 ||  $KATELLO_VERSION == 4.1 ]]; then
+    skip "pulpcore import/export tests are not available on Katello versions older than 4.3"
```

Comment:
This still says import/export.  Also I wonder if these `tSkipIfOlderThanBlah` tests are going to start piling up.